src/corehost/cli/ijwhost/AMD64/bootstrap_thunk.h

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#ifndef IJW_BOOTSTRAP_THUNK_H
+#define IJW_BOOTSTRAP_THUNK_H
+
+#if !defined(_TARGET_AMD64_)
+#error "This file should only be included on amd64 builds."
+#endif
+
+#include "pal.h"
+#include "corhdr.h"
+
+extern "C" void start_runtime_thunk_stub();
+
+#include <pshpack1.h>
+class bootstrap_thunk
+{
+private:
+    // 49 BA 78 56 34 12 78 56 34 12 mov         r10,1234567812345678h
+    // 49 BB 34 12 34 12 34 12 34 12 mov         r11,1234123412341234h
+    // 41 FF E3                      jmp         r11
+
+    static BYTE             s_mov_r10[2];
```

Comment:
I would remove these static's from the class. Also, they should be marked `const`.